### PR TITLE
fix sabre-lite link

### PR DESCRIPTION
**Pull request by lsf37**
Thanks to @robs-cse for pointing out the new link.

### CODE REVIEW

File: Hardware/sabreLite.md [hunk 1]
```diff
@@ -182,4 +182,4 @@ This assumes that the U-Boot version above is installed in SPI flash.
 
 ## Running seL4 test
 
-{% include sel4test.md %}
\ No newline at end of file
+{% include sel4test.md %}
```
Comment (Xaphiosis) [inline Hardware/sabreLite.md:185]: interesting, what changed here?

**Review by Xaphiosis — APPROVED** [review]

File: Hardware/sabreLite.md (hunk 1 above)
Comment (lsf37) [reply Hardware/sabreLite.md:185]: I suspect my editor removed trailing whitespace, let me check

Comment (lsf37) [reply Hardware/sabreLite.md:185]: Oh, no my editor wasn't involved, this was via the GitHub interface. So I'm not sure actually.

Comment (lsf37) [reply Hardware/sabreLite.md:185]: Looking at it in `git diff`, it added a newline at the end of the last line, which I approve of.

**Pull request merged**